### PR TITLE
central and lateral incisor tooth

### PR DESCRIPTION
**Pull request by meghalithic**
changing definition and EQ
issue #2719 

see discussion [here](https://docs.google.com/document/d/1XvMbNvr0FEsdqGhg79BYCYEHSqUxRHMcvhbGizEAht8/edit) on 28/11/2022.

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -168161,6 +168162,7 @@ id: UBERON:0016455
 name: upper lateral secondary incisor tooth
 subset: pheno_slim
 synonym: "maxillary lateral secondary incisor tooth" EXACT [FMA:55724]
+synonym: "secondary upper second incisor tooth" EXACT []
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:168165]: Is there a dbxref to support calling this the "second" incisor tooth?

**Review by bvarner-ebi — COMMENTED** [review]
Hi @megbalk, I don't see any edits that would "break" the ontology in this PR, but from a content perspective I recommend seeking a review from someone qualified in this subject matter.

Some of the terms do not have text definitions and the added synonyms do not have dbxrefs to check.

Regarding, "... phylogenetically number #" in the text definitions... after a cursory search online, I could not find an explanation of a phylogenetic numbering system used in this context. This may be due to a lack of domain knowledge, which is why I recommend seeking a review from someone with subject matter expertise.

The annotation property ['tooth number'](http://purl.obolibrary.org/obo/UBPROP_0000112) mentions an ancestral count and a count along the A-P axis. I don't know if the most anterior tooth in a set is always ancestrally the first to appear. Hopefully someone with domain knowledge could help.

In summary, I recommend:

- adding text definitions where they are missing
- adding dbxrefs to support the synonyms
- seeking someone with subject matter expertise in dentition to review the content

If there is a reference that you can share that describes phylogenetically numbering from an anatomical perspective, I'm happy to take a look and do another review.

**Comment by megbalk** [comment]
@cmungall @RDruzinsky could I get some help here?

**Comment by anitacaron** [comment]
#gogoeditdiff

**Comment by anitacaron** [comment]
#gogoeditdiff

**Comment by anitacaron** [comment]
@wdduncan could you please review this PR?

**Comment by anitacaron** [comment]
@wdduncan you can check the nice diff view [here](https://github.com/obophenotype/uberon/pull/2757#issuecomment-1874470012)

**Comment by wdduncan** [comment]
This seems fine to me.

**Review by wdduncan — APPROVED** [review]
